.gitignore

```diff
@@ -4,15 +4,14 @@ coverage/
 coverage.json
 deployments/
 dist/
-typechain-types
+typechain-types/
+venv/
```

Comment:
we do python now?

Reply:
The certora verification uses pip for installing the Certora CLI.

Reply:
I am not a Python expert, but it doesn't seem like the virtual env gets initialised anywhere in the script?

Reply:
No it does not - I use it locally, since our different projects use different versions of `certora-cli`, so installing it globally isn't great. I basically follow the instructions from:

https://github.com/safe-global/safe-locking?tab=readme-ov-file#formal-verification